mod/01_core/set/all/fetch.rb

```diff
@@ -55,7 +55,9 @@ def fetch mark, opts={}
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
       else
-        return unless !opts[:skip_virtual] && card.virtual?
+        card.include_set_modules unless opts[:skip_modules]  # need to load modules here to call the right virtual? method 
+        opts[:skip_modules] = true                           # don't load twice
+        return if opts[:skip_virtual] || !card.virtual?
```

Comment:
I overwrite raw_content and include Set::Type::Pointer. That's what makes the card behave as a virtual card. It would work with a dummy *followers+*right+*structure but the card doesn't need a structure.